internal/flink/command_artifact_describe.go

```diff
@@ -0,0 +1,37 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *command) newDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe <id>",
+		Short: "Describe a Flink UDF artifact.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.describe,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Describe Flink UDF artifact",
+				Code: "confluent connect artifact describe ccp-123456",
```

Comment:
```suggestion
				Code: "confluent flink artifact describe ccp-123456",
```

Reply:
Why does the resource start with "ccp-" if this is a Flink command?

Reply:
as we reusing custom connector plugin api, plugin id will start with `ccp-` for EA

Reply:
Interesting, can we fix the command name?